CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 12.0.1
```

Comment:
Let's also version bump [`logstash-output-elasticsearch.gemspec`](https://github.com/logstash-plugins/logstash-output-elasticsearch/blob/main/logstash-output-elasticsearch.gemspec#L3) too.
Or are you going to absorb this into #1199?

Reply:
I was thinking of releasing both in 12.0.1 as well as backporting to 11. I'll wait to merge this until i've got a clear path. Thanks.